.eslintrc.json

```diff
@@ -13,6 +13,8 @@
       "functions": "never"
    }],
    "no-confusing-arrow": 0,
-   "jsx-a11y/href-no-hash": "off"
+   "jsx-a11y/href-no-hash": "off",
+   "no-underscore-dangle": "off",
```

Comment:
The main thing behind it is the testing, because we shouldn't test private functions, if we are not going to use underscore, we need another syntax definition for private, but the underscore is the most common one.

Reply:
If the 'private' functions in question would be defined in the module but not exported with the class they can still be somehow private, as in, anything that imports JsonRpc wouldn't have access to them, though with JsonRpc still having access to them. Since _scanTransactions actually only needs one property and other _functions only use 'this' to call other _functions, this can be done by passing the needed property 'address' as an argument.

Reply:
this would remove the need for an underscore since you can't access those functions outside of the module itself.